src/generator/AutoRest.CSharp/Templates/Rest/Client/MethodTemplate.cshtml

```diff
@@ -353,12 +353,12 @@ else
         @:{
             if (Model.DefaultResponse.Body.IsPrimaryType(KnownPrimaryType.Stream))
             {
-            @:@(Model.DefaultResponse.Body.AsNullableType()) _errorBody = await _httpResponse.Content.ReadAsStreamAsync().ConfigureAwait(false);
+            @:@(Model.DefaultResponse.Body.AsNullableType(Model.IsXNullableReturnType)) _errorBody = await _httpResponse.Content.ReadAsStreamAsync().ConfigureAwait(false);
```

Comment:
Careful: this seems to be the error body, which is orthogonal to the "return type" body! So this does not belong here.